### ISSUE TITLE
chronyd should be disabled on vagrant boxes

### ISSUE THREAD

**Issue by ethanmdavidson** [oracle/vagrant-boxes#132]
The ol7-latest box has chronyd enabled and running. However, with vagrant it should be disabled because time should be synchronized with the host via virtualbox guest additions. 

Can you please rebuild the box with chronyd disabled?

(This might also apply to some of the other boxes, but ol7 is the only one I have checked)

**Comment by gvenzl** [comment]
@scoter-oracle, @totalamateurhour, does that requirement make sense? I have no technical of `chronyd` and what the impact would or could be.

**Comment by scoter-oracle** [comment]
The question is: does chronyd creates any issue on the box ?
Those boxes can be also exported and, optionally, reused on other hypervisors.

**Comment by ethanmdavidson** [comment]
I should clarify that the original reason I requested this was because the security guy at my company didn't like all the traffic going to random NTP servers. In other words, there was no specific issue caused by chronyd, but I thought it should be removed anyway because guest additions was synchronizing the time so chronyd was just creating unnecessary traffic. However, I didn't think about the impact on other hypervisors. 

I've worked around this by simply disabling ntp as part of the box provisioning.
